Atomic/containers.py

```diff
@@ -194,3 +213,38 @@ def _filter_include_container(self, container_info):
                 return False
 
         return True
+
+    def delete(self):
+        results = 0
+        sys_targets=[]
+        docker_targets=[]
+        if self.args.all:
+            for c in self.get_containers():
+                if c["Type"] == "system":
+                    sys_targets.append(c["Id"])
+                else:
+                    docker_targets.append(c["Id"])
+        else:
+            for c in self.args.containers:
+                if self._inspect_container(c):
+                    docker_targets.append(c)
+                else:
+                    sys_targets.append(c)
```

Comment:
we look up system containers first then try the docker version.  I think here we should do:
```
if self.syscontainers.get_system_container_checkout(self, name):
    sys_targets.append(c)
else:
    docker_targets.append(c)
```